src/Core/Domain/Order/QueryResult/OrderCustomerForViewing.php

```diff
@@ -114,6 +128,8 @@ public function __construct(
         $this->validOrdersPlaced = $validOrdersPlaced;
         $this->privateNote = $privateNote;
         $this->isGuest = $isGuest;
+        $this->ape = $ape;
+        $this->siret = $siret;
```

Comment:
Consider casting these to string so that you can avoid mixed return types

Reply:
Fixed.